pkg/operator/configobservation/interfaces.go

```diff
@@ -13,6 +13,7 @@ type Listers struct {
 	BuildConfigLister  configlistersv1.BuildLister
 	ConfigMapLister    corelistersv1.ConfigMapLister
 	NetworkLister      configlistersv1.NetworkLister
+	FeatureGateLister_ configlistersv1.FeatureGateLister
```

Comment:
Accidental underscore?
```suggestion
	FeatureGateLister  configlistersv1.FeatureGateLister
```